ci/docs.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Helper script to build documentation using Sphinx
+# NOTE: Should only be run on archlinux:latest 
+
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+set -eux
+
+show-green () {
+    echo -e "${GREEN} ==> ${1} ${NC}" 
+}
+
+# build package from the AUR (arch user repository)
+if command -v makepkg &> /dev/null
+then
+    show-green "Cloning sugar-toolkit-gtk3-git from AUR"
+    mkdir .cache
+    cd .cache 
+    git clone https://aur.archlinux.org/sugar-toolkit-gtk3-git.git
+    cd sugar-toolkit-gtk3-git
+    makepkg -sif --noconfirm 
+    show-green "Installing optional dependencies"
+    yes | sudo pacman -S --needed $(cat .SRCINFO| grep 'optdepends' | grep -o '= .*:' | sed 's,= ,,g' | sed 's,:,,g') --noconfirm
```

Comment:
same here: yes is probably unnecessary.

Reply:
See https://unix.stackexchange.com/a/293537